README.md

```diff
@@ -63,17 +63,72 @@ to write the model out.
+and converted to Java by Next Century.  This version of the validator accepts multiple
+ontology files, can validate against NIST requirements (restricted AIF), and can
+validate N files or all files in a specified directory.
+
+### Running the AIF validator
+To run the validator from the command line, run `target/appassembler/bin/validateAIF`
+with a series of command-line arguments (in any order) honoring the following usage:  <br>
+Usage:  <br>
+`validateAIF { --ldc | --program | --ont FILE ...} [--nist] [-h | --help] {-f FILE ... | -d DIRNAME}`  <br>
+Options:  <br>
+`--ldc` validate against the LDC ontology  <br>
+`--program` validate against the program ontology  <br>
+`--ont FILE ...` validate against the OWL-formatted ontolog(ies) at the specified filename(s)  <br>
+`--nist` validate against the NIST restrictions  <br>
+`-h, --help` This help and usage text  <br>
+`-f FILE ...` validate the specified file(s) with a .ttl suffix  <br>
+`-d DIRNAME` validate all .ttl files in the specified directory  <br>
+Either a file (-f) or a directory (-d) must be specified (but not both).  <br>
+Exactly one of --ldc, --program, or --ont must be specified.  <br>
+Ontology files can be found in `src/main/resources/com/ncc/aif/ontologies`:
+- LDC (LO): `SeedlingOntology`
+- Program (AO): `EntityOntology`, `EventOntology`, `RelationOntology`
+
+### Validator return values
+Return values from the command-line validator are as follows:
+* `0 (Success)`.  There were no validation (or any other) errors.
+* `1 (Validation Error)`.	All specified files were validated but at least one failed validation.
+* `2 (Usage Error)`.  There was a problem interpreting command-line arguments.  No validation was performed.
+* `3 (File Error)`.  There was a problem reading one or more files or directories.  Validation may have been performed on a subset of specified KBs.  If there is an error loading any ontologies or SHACL files, then no validation is performed.
+
+### Running the validator in code
+To run the validator programmatically in Java code, first use `ValidateAIF.create()`
+to create a validator object, then call one of the public `validateKB()` methods.
+`create()` accepts a set of domain ontology CharSources and several flags.  See the JavaDocs.
+
+Note: the original `ValidateAIF.createForDomainOntologySource()` method remains for backward compatibility.
+
+### Differences from the legacy validator
+The AIF Validator bears certain important differences from the previous version of
+the validator (still currently available-- see *Running the legacy validator* below).
+* The validator no longer accepts a parameter file to specify, essentially,
+its program arguments.  Instead, it takes all arguments as command-line options.
+* The validator no longer ensures that confidences are between 0 and 1.
+* The validator will now only validate files with the `.ttl` extension.
+* The validator returns a variety of return codes (see above).
+
+### Validation duration
```

Comment:
Re: Validation duration: Interesting information but...

- doesn't belong in the README - too many variables. We can put this in confluence.
- We need to remove references to algorithms from TA teams.  If we had more hard numbers with processor speeds, we could put this in the AIDA Collaboration confluence but just using generic information (like TA1/TA2/TA3, size, duration) without specific references to performer algorithms.

Reply:
Removed, and moved placeholder to Confluence page.